_unittest/test_41_3dlayout_modeler.py

```diff
@@ -249,9 +249,36 @@ def test_10_create_padstack(self):
 
     def test_11_create_via(self):
         via = self.aedtapp.modeler.create_via("My_padstack2", x=0, y=0, name="port_via")
-        assert type(via) is str
-        via = self.aedtapp.modeler.create_via("My_padstack2", x=10, y=10, name="Via123", netname="VCC")
-        assert via == "Via123"
+        assert isinstance(via, str)
+        assert self.aedtapp.modeler.vias[via].name == via
```

Comment:
It is just to check that the string returned when creating the via is correct.
```suggestion
        assert self.aedtapp.modeler.vias[via].name == via == "port_via"
```